src/snoowrap.js

```diff
@@ -238,6 +238,87 @@ const snoowrap = class snoowrap {
     });
   }
 
+  static get grantType () {
```

Comment:
Could we also add JSDoc for this?

Reply:
Done. But on closer inspection, doesn't this belong to `constants.js`?

Something like:

```
export const GRANT_TYPE = {
    CLIENT_CREDENTIALS: 'client_credentials',
    INSTALLED_CLIENT: 'https://oauth.reddit.com/grants/installed_client'
}
```

Only usage I can find for this is in the tests.

Reply:
Do you know if it shows up in the docs? You can run `npm run build-docs` and then open index.html into browser to check. I did this for ModmailConversation but it doesn't work that well on JSDoc for some reason: https://github.com/not-an-aardvark/snoowrap/blob/master/src/objects/ModmailConversation.js#L3-L14

I'm unsure about putting it into `constants.js`, I think it would be a good idea. @not-an-aardvark thoughts?